crates/ruma-macros/src/api.rs

```diff
@@ -6,13 +6,13 @@ use once_cell::sync::Lazy;
 use proc_macro2::Span;
 use serde::{de::IgnoredAny, Deserialize};
 
-mod api_metadata;
+// mod api_metadata;
```

Comment:
@jplatte, I deactivated these modules because it complains that it is dead code. It seems that it is indeed the case, I am wondering why it wasn't complaining before? Should we remove them?

Reply:
We had a similar issue with Conduit, but that was for unused fields on a tuple struct, which was recently merged into `dead_code`. I don't see anything about modules in the changelog tho. 🤔

Reply:
It was actually complaining that _all_ the fields of a struct were unused. So I ended up testing by removing the whole module.

Reply:
Sure, remove it.

Reply:
Removed